pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -66,31 +66,30 @@ const wxChar* Panels::SpeedHacksPanel::GetEEcycleSliderMsg( int val )
 	return L"Unreachable Warning Suppressor!!";
 }
 
-const wxChar* Panels::SpeedHacksPanel::GetVUcycleSliderMsg( int val )
+const wxChar* Panels::SpeedHacksPanel::GetEECycleSkipSliderMsg( int val )
 {
 	switch( val )
 	{
 		case 0:
 		{
 			const wxColour DarkSeaGreen = wxColour(14, 158, 19);
-			m_msg_vustealer->SetForegroundColour(DarkSeaGreen);
-			return pxEt(L"0 - Disables VU Cycle Stealing.  Most compatible setting!");
+			m_msg_eeSkip->SetForegroundColour(DarkSeaGreen);
+			return pxEt(L"Disables EE Cycle Skipping. Most compatible setting.");
 		}
 		case 1:
 		{
-			m_msg_vustealer->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"1 - Mild VU Cycle Stealing.  Lower compatibility, but some speedup for most games.");
+			m_msg_eeSkip->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"Mild EE Cycle Skipping. Lower compatibility, may cause variable framerate games to reduce internal framerate. Helps some games with VU starvation problems run at full speed. E.g. Shadow of the Colossus.");
 		}
 		case 2:
 		{
-			m_msg_vustealer->SetForegroundColour(wxColour(L"Red"));
-			return pxEt(L"2 - Moderate VU Cycle Stealing.  Even lower compatibility, but significant speedups in some games.");
+			m_msg_eeSkip->SetForegroundColour(wxColour(L"Red"));
+			return pxEt(L"Moderate EE Cycle Skipping. Even lower compatibility, variable FPS games will reduce internal FPS. Games with VU starvation problems will gain emulation speed, others will likely start to slow down.");
```

Comment:
The other strings now use framerate instead of FPS. This string should also be changed for consistency. It may be boring to have strings consistent, but IMO it makes the string easier to understand and also translate since there are fewer distractions (e.g. "Why is it FPS here and framerate elsewhere? Does it mean something different?") and it's easier to compare the differences in the strings.

Also, could `emulation speed` be replaced with just `speed`?